src/main/java/de/hpi/bpt/argos/api/entityType/EntityTypeEndpoint.java

```diff
@@ -0,0 +1,90 @@
+package de.hpi.bpt.argos.api.entityType;
+
+import de.hpi.bpt.argos.core.Argos;
+import de.hpi.bpt.argos.util.RestUriUtil;
+import spark.Request;
+import spark.Response;
+
+import java.util.Objects;
+
+/**
+ * This interface represents the endpoint to receive entity types.
+ */
+public interface EntityTypeEndpoint {
+    String ENTITY_TYPE_BASE_URI = String.format("%1$s/api/entitytype", Argos.getRoutePrefix());
+
+    /**
+     * This method returns the hierarchy overview of all entityTypes.
+     * @param request - Spark defined parameter containing request object
+     * @param response - Spark defined parameter containing response object
+     * @return - returns a JSON String of the hierarchy
+     */
+    String getEntityTypeHierarchy(Request request, Response response);
+
+    /**

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=checkstyle%3AJavadoc_Style_BP2016)